resources/schema.py

```diff
@@ -268,6 +267,10 @@ class Meta:
         "\n * Filter `forApplication` combined with either dimension (width, length) filter",
     )
 
+    @classmethod
+    def get_node(cls, info, id):
+        return Harbor.objects.language(get_language()).get(pk=id)
```

Comment:
what made you add this?

Reply:
We were fetching the harbor using the `language()` queryset on the [`UpdateHarborMutation`](https://github.com/City-of-Helsinki/berth-reservations/pull/184/files#diff-09f1721fc193a5a442fee1cf24f5bd49L597)
```python
lang = get_language()
harbor = get_node_from_global_id(info, input.pop("id"), only_type=HarborNode)

try:
    harbor = Harbor.objects.language(lang).get(pk=id)
```

But since we don't have direct access to the queryset anymore using the `get_node_from_global_id`, you can't explicitly call the language filter.

However, `get_node_from_global_id` internally calls `get_node`, so, by overriding `get_node` and forcing the query to use `language` we ensure that it's always returning the one with the correct language even if we can manually get it.

Reply:
as locally tested, Django's `LocaleMiddleware` can handle the language itself, no need for us to pass it explicitly in resolvers.

Reply:
you can probably delete other lines with `get_language` from this file 🙂